generators/server/templates/src/test/java/package/config/TestContainersSpringContextCustomizerFactory.java.ejs

```diff
@@ -42,6 +42,9 @@ public class TestContainersSpringContextCustomizerFactory implements ContextCust
 <%_ if (messageBrokerKafka) { _%>
     private static KafkaTestContainer kafkaBean;
 <%_ } _%>
+<%_ if (messageBrokerPulsar) { _%>
+    private static PulsarTestContainer pulsarBean;
+<%_ } _%>
```

Comment:
```suggestion
```

Reply:
The other beans don't have these skipped lines.
It should be kept consistent.

Reply:
Related to https://github.com/jhipster/generator-jhipster/pull/21563#discussion_r1149131359.

Reply:
Oh, sorry. I misunderstood the change.

Reply:
done